eng/common/testproxy/test-proxy-docker.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
+    rootFolder: '$(Build.SourcesDirectory)'
+
+steps:
+  - pwsh: |
+        . $(Build.SourcesDirectory)/eng/common/scripts/common.ps1
```

Comment:
These lines might be worth adding an script to execute. Especially if you plan to have devs do these steps locally.

Reply:
I'm seeing an integration issue that's going to force this, so good point!